graderx/graders/manager.py

```diff
@@ -91,7 +91,8 @@ def add_submissions(course_name, lab, submissions_file):
     All the possibly returned modules will have a add_submissions function that will be invoked here
     """
     course_grader = select_course_grader(course_name)
-    course_grader.add_submissions(course_name, lab, submissions_file)
+    submission_key = course_grader.add_submissions(course_name, lab, submissions_file)
```

Comment:
does the `add_submissions(course_name, lab, submissions_file)` add a single submission file? (so the function should be names `add_submission` without an *s*) or the `submissions_file` variable contains multiple submissions? (so it should be names `submission_files`)

Reply:
it adds multiple submissions, but 'submissions_file' is a compressed file containing multiple submissions, is it still more descriptive to make it submission_files?

Reply:
I think submissions_compressed_file could be less confusing

Reply:
I think submissions_compressed_file can be more descriptive

Reply:
yes, or decompress first then pass the uncompressed files to the `course_grader` since it's not its responsibility to decompress files.